pypeit/scripts/show_2dspec.py

```diff
@@ -64,11 +64,15 @@ def show_trace(specobjs, det, viewer, ch):
         trace = specobjs[kk]['TRACE_SPAT']
         obj_id = specobjs[kk].NAME
         maskdef_objname = specobjs[kk].MASKDEF_OBJNAME
+        force_extr_flag = specobjs[kk].FORCE_EXTRACT
         if maskdef_objname is not None:
             trc_name = '{}     OBJNAME:{}'.format(obj_id, maskdef_objname)
         else:
             trc_name = obj_id
-        display.show_trace(viewer, ch, trace, trc_name, color='orange') #hdu.name)
+        if force_extr_flag is not None and force_extr_flag is True:
```

Comment:
I like this, but again consistency with manual objects is desired.

Reply:
We might want yet another color then.

Reply:
I added a flag for manual extraction and it will have a different color in ginga.